### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by sonal251001**
# Related Issue
Fixes:  Changes in README.md

# Prosposed Changes
- change 1 : Donate education changed to provide education
- change 2 : Added 'was' 'the' 'were' in **achievements section**

# Additional Info
- any additional information or context
Hey @garg3133 I'm not able to open the development.py file as @tmo66 told . I would request you to merge this only after reviewing.
# Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Comment by garg3133** [comment]
This looks good! Thank you, @sonal251001! But you left one part of the issue you were assigned "To actually send emails to real users, comment line 27 and uncomment line 28 in Jagrati/settings/development.py (This should be line 30 and line 31)". Can you please do this also?

For this part, you need to find the above mentioned lines in the README file (which is present in the **Setting-up the project** section) and replace 27 & 28 with 30 & 31 respectively.

**Comment by sonal251001** [comment]
I'm really sorry I'm not getting that.

**Comment by garg3133** [comment]
@sonal251001 In Readme's **Setting-up the project** section, replace the below point:

* `EMAIL_HOST_USER` and `SENDER_EMAIL` is the email address of your Gmail account from which you want to send emails (By default, Django will output email contents in console. To actually send emails to real users, comment line 27 and uncomment line 28 in `Jagrati/settings/development.py`).

with:

* `EMAIL_HOST_USER` and `SENDER_EMAIL` is the email address of your Gmail account from which you want to send emails (By default, Django will output email contents in console. To actually send emails to real users, comment line 30 and uncomment line 31 in `Jagrati/settings/development.py`).

(Notice the change in line numbers mentioned in the above point, 27 & 28 is replaced by 30 & 31.

**Comment by sonal251001** [comment]
I have done it now please check and let me know if it works.

File: README.md [hunk 1]
```diff
@@ -10,16 +10,16 @@
...
 
 **Our Achievements:**
-- 1 student selected in Jawahar Navodaya Vidyalaya in 2018.
-- 1 student selected in Jawahar Navodaya Vidyalaya in 2019.
-- 2 students selected in Jawahar Navodaya Vidyalaya in 2020.
-- 4 students cleared entrance examination for Eklavya Model Residential School (EMRS) in 2020.
-- 4 students cleared entrance examination for Gyanodaya Vidyalaya in 2020.
+- 1 student was selected in Jawahar Navodaya Vidyalaya in 2018.
+- 1 student was selected in Jawahar Navodaya Vidyalaya in 2019.
+- 2 students were selected in Jawahar Navodaya Vidyalaya in 2020.
```
Comment (garg3133) [inline README.md:20]: One more small change, I think in place of `in` in above three lines, `for` should be present.

**Comment by garg3133** [comment]
Also, you can link this pull request with the issue it is fixing using the linkifier, `Fixes: #issue-no`.  You can place this linkifier anywhere in the issue description. It helps automatically close the issue once a pull request is merged.

**Comment by sonal251001** [comment]
I have done that please review it . Fixes: #issue-no https://github.com/garg3133/JagratiWebApp/issues/156

**Comment by garg3133** [comment]
Merged! 🎉 

Thanks, @sonal251001 for you contribution 🚀

**Comment by sonal251001** [comment]
Hey @garg3133 I'm a GSSoC participant it would be great  if you could add relavent labels so that my PR is counted. Could you please see to it?

**Comment by sonal251001** [comment]
Hey @garg3133  I'm also a GSSoC participant so can you please add relavent labels like gssoc21 and the level of problem so that my PR could be counted.

**Comment by garg3133** [comment]
Ohh sorry, I must have forgotten that!

**Pull request merged**